licenses/BSL.txt

```diff
@@ -36,7 +36,7 @@ ADDITIONAL USE GRANT:   You may use the Licensed Work in accordance with the
                         3. using the Licensed Work to create a work or solution
                         which competes (or might reasonably be expected to
                         compete) with the Licensed Work.
-CHANGE DATE:            2026-18-02
+CHANGE DATE:            2026-27-05
```

Comment:
```suggestion
CHANGE DATE:            2026-27-04
```

Don't we have to update the version in CMake also?

Reply:
CMake version (`MEMGRAPH_OVERRIDE_VERSION`) is only updated on patch releases if that's what you are referring to